h/views/api/groups.py

```diff
@@ -5,6 +5,7 @@
 from pyramid import security
 from pyramid.httpexceptions import HTTPNoContent, HTTPBadRequest, HTTPNotFound
 
+from h.auth.util import client_authority
```

Comment:
Discussion for another PR, but I'd like to bring up the notion of having `client_authority` available on the request (a la `default_authority`). We've got several places in view-level code now where I'm having to import an `auth` module utility function and it is starting to feel funky.